MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,1085 @@
+<td>IsReadOnly</td>
+<td>Gets or sets a value indicating whether editors are enabled for editing.</td>
+</tr>
+
+<tr>
+<td>{{'[CommitMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_CommitMode)'| markdownify }}</td>
+<td>CommitMode</td>
+<td>Gets or sets the value that determines when to commit values that users enter in the data form’s editors to the bound data object.</td>
+</tr>
+
+<tr>
+<td>{{'[ValidationMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_ValidationMode)'| markdownify }}</td>
+<td>ValidationMode</td>
+<td>Gets or sets a value that denotes when the user input should be validated based on IDataErrorInfo and ValidationAttributes.</td>
+</tr>
+
+<tr>
+<td>{{'[Items](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_Items)'| markdownify }}</td>
+<td>Items</td>
+<td>Gets or sets the value which used to set the DataFormViewItem collection in the SfDataForm.</td>
+</tr>
+
+<tr>
+<td>{{'[ColumnCount](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_ColumnCount)'| markdownify }}</td>
+<td>ColumnCount</td>
+<td>Gets or sets the value representing the number of editors to be arranged per row in the data form.</td>
+</tr>
+
+<tr>
+<td>{{'[SourceProvider](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_SourceProvider)'| markdownify }}</td>
+<td>ItemsSourceProvider</td>
+<td>Gets or sets the value to get the source for picker type editors.</td>
+</tr>
+
+<tr>
+<td>{{'[AutoGenerateItems](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_AutoGenerateItems)'| markdownify }}</td>
+<td>AutoGenerateItems</td>
+<td>Gets or sets a value indicating whether auto generate the DataFormItems in the data form.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>EditorTextStyle</td>
+<td>Gets or sets the style of editor text, that used to customize the text color, font, font size, font family and font attributes.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>LabelTextStyle</td>
+<td>Gets or sets the style of label text, that used to customize the text color, font, font size, font family and font attributes.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>ErrorLabelTextStyle</td>
+<td>Gets or sets the style of error label text, that used to customize the text color, font, font size, font family and font attributes.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>ValidMessageLabelTextStyle</td>
+<td>Gets or sets the style of validation label text, that used to customize the text color, font, font size, font family and font attributes.</td>
```

Comment:
<td>Gets or sets the style of the validation label text used to customize the text color, font size, font family, and font attributes.</td>